xdsl/transforms/convert_onnx_to_linalg.py

```diff
@@ -93,7 +93,7 @@ def make_unique_name(self):
 
     @op_type_rewrite_pattern
     def match_and_rewrite(self, constant: onnx.Constant, rewriter: PatternRewriter, /):
-        attr_value = list(constant.attributes.values())[1]
+        attr_value = constant.value
```

Comment:
get_value()?

Reply:
> get_value()?
helper function? you mean

Reply:
sorry, i am confused what you mean

Reply:
Yep, whatever the type of the value, we should create a global with it. This code only gets the default `dense` of floats, but if it's ints it'll break

Reply:
of so parse the attr_value type into the global?